war/src/main/js/api/search.js

```diff
@@ -0,0 +1,11 @@
+/**
+ * @param {string} searchTerm
+ */
+async function search(searchTerm) {
+  const address = document
+    .getElementById("page-header")
+    .dataset.searchUrl.escapeHTML();
+  return fetch(`${address}?query=${searchTerm}`);
```

Comment:
🐛 
```suggestion
  return fetch(`${address}?query=${encodeURIComponent(searchTerm)}`);
```

Tested locally with just `a&b`, only `a` is received.
The & will "escape" the query part and create another property=value in the URL. I don't think there is a security impact but more a bug.